lib/noid/minter.rb

```diff
@@ -46,6 +47,14 @@ def valid?(id)
       template.valid?(id)
     end
 
+    ##
+    # Returns the number of identifiers remaining in the minter
+    # @return [Fixnum]
+    def remaining
+      return 'unlimited' if unbounded?
```

Comment:
Even better. I hated returning a string but didn't know Ruby had a constant for infinity, Change coming.